tests/contrib/operators/test_dingding_webhook_operator.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+from airflow import DAG, configuration
+from airflow.contrib.operators.dingding_webhook_operator import DingdingWebhookOperator
+from airflow.utils import timezone
+
+DEFAULT_DATE = timezone.datetime(2017, 1, 1)
+
+
+class TestDingdingWebhookOperator(unittest.TestCase):
+    _config = {
+        'http_conn_id': 'dingding_webhook_default',
+        'webhook_token': 'manual_token',
+        'message_type': 'text',
+        'message': 'Airflow dingding webhook test',
+        'at_mobiles': ['123', '456'],
+        'at_all': False
+    }
+
+    def setUp(self):
+        configuration.load_test_config()
+        args = {
+            'owner': 'airflow',
+            'start_date': DEFAULT_DATE
+        }
+        self.dag = DAG('test_dag_id', default_args=args)
+
+    def test_execute(self):
```

Comment:
This test only checks the constructor. The execute method is not called.

Reply:
I do that because many of operator check just check constructor, like `test_slack_webhook_operator.py` and `test_spark_jdbc_operator.py`, I think they do that due to operator is quick simple, and all the test could check in hook?
what do you think?

Reply:
Add test, thanks.